workflows/testing/tester.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+import tempfile
+import urllib.request
+from test_workflow.manifest import BuildManifest
+from test_workflow.test_cluster import LocalTestCluster
+from test_workflow.git import GitRepository
+from test_workflow.integ_test import IntegTestSuite
+
+if (len(sys.argv) < 2):
+    print("Usage: tester.py /path/to/manifest")
+    exit(1)
+
+with tempfile.TemporaryDirectory() as work_dir:
+    os.chdir(work_dir)
+
+    # Download the manifest
+    with urllib.request.urlopen(sys.argv[1]) as response:
+        text = response.read()
+        manifest = BuildManifest.from_text(text)
+
+    # Spin up a test cluster
+    cluster = LocalTestCluster(manifest.bundle_location)
+
+    # For each component, check out the git repo and run `integtest.sh`
+    for component in manifest.components():
+        if (component.name() == 'opensearch'):
```

Comment:
Why add this special case?

Reply:
Because we don't have integtest.sh for this repo. It's a big repo, so checking out a ton of code that we won't use is a substantial waste of time.

Reply:
> checking out a ton of code that we won't use

Hmm... is this worth doing a sparse checkout to make it faster? What kind of performance should we expect?

Reply:
Changed to use `depth 1`.